### PR TITLE
Adding shapes

### PR DESCRIPTION
**Pull request by gregdolley**

### CODE REVIEW

File: __tests__/shapes.test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const PPTX = require('../index.js');
+const fs = require('fs');
+//const path = require('path');
```
Comment (heavysixer) [inline __tests__/shapes.test.js]: can we delete this if we don't need it?

File: __tests__/shapes.test.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const tmpDir = `${__dirname}/tmp`;
+
+describe('Presentation Module', () => {
+    beforeAll(() => {
+        //prepareTmpDir(tmpDir);
```
Comment (heavysixer) [inline __tests__/shapes.test.js]: if we don't need the `beforeAll` or `afterAll` can we delete it?

File: __tests__/smoke.test.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 const PPTX = require('../index.js');
 describe('test essentials', () => {
     test('true', () => {
-        expect(Object.keys(PPTX)).toEqual(['Presentation', 'Slide']);
+        expect(Object.keys(PPTX)).toEqual(['Presentation', 'Slide', 'ShapeTypes']);
```
Comment (heavysixer) [inline __tests__/smoke.test.js]: Maybe just `Shapes`?

File: lib/pptx.js [hunk 4]
```diff
@@ -3,4 +3,9 @@ let { Slide } = require('./slide');
 module.exports = {
     Presentation: require('./presentation'),
     Slide: Slide,
+    ShapeTypes: require('./shape-types').ShapeTypes, // TODO: I don't like this here...
 };
+
+// Some aliases I like for shape types (Greg D.):
+module.exports.ShapeTypes.TRIANGLE = module.exports.ShapeTypes.ISOSCELES_TRIANGLE;
```
Comment (heavysixer) [inline lib/pptx.js]: Can't you define these aliases inside the ShapeType class itself?

File: __tests__/shapes.test.js (hunk 1 above)
Comment (gregdolley) [reply __tests__/shapes.test.js]: It's for the dir cleaning function which is temporarily commented out. Sometimes I run that test on it's own.

File: __tests__/shapes.test.js (hunk 2 above)
Comment (gregdolley) [reply __tests__/shapes.test.js]: Yeah, I'll delete them.

File: lib/pptx.js (hunk 4 above)
Comment (gregdolley) [reply lib/pptx.js]: Originally I had. Then I didn't like the idea of modifying officegen's original source file.

File: __tests__/smoke.test.js (hunk 3 above)
Comment (gregdolley) [reply __tests__/smoke.test.js]: Sounds to similar to the "Shape" class though...

File: __tests__/shapes.test.js (hunk 1 above)
Comment (gregdolley) [reply __tests__/shapes.test.js]: Nevermind. I deleted the "scrubber" functions anyway. It's easier this way.

File: __tests__/shapes.test.js (hunk 2 above)
Comment (gregdolley) [reply __tests__/shapes.test.js]: Done.

File: lib/pptx.js (hunk 4 above)
Comment (heavysixer) [reply lib/pptx.js]: it's our source file now :-)

**Pull request merged**